app/modules/submissions/reducers/submissions.js

```diff
@@ -78,6 +30,7 @@ const submission = (state, action) => {
     } else {
       return state
     }
+  // Move out to global state
```

Comment:
:art: If these are `TODO` comments, let's make it really clear to differentiate from other comments